src/main_WorldViewer.cpp

```diff
@@ -14,9 +14,27 @@
 #include <original-content/OriginalGameFiles.hpp>
 #include <original-content/VirtualFileSystem.hpp>
 
+class Config : public REGoth::EngineConfig
+{
+
+public:
+
+  virtual void registerCLIOptions(cxxopts::Options& opts) override
+  {
+    opts.add_options()
+      ("w,world", "World name to load", cxxopts::value<bs::String>(world), "[NAME]")
```

Comment:
We're not loading a "World name" here, are we? :wink: 

How about `Name of the world to load`?

Reply:
Changed to `Name of the world to load`. Brain fart